src/theme/BlogPostItem/index.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { DiscussionEmbed } from 'disqus-react'
+import BlogPostItem from '@theme-original/BlogPostItem';
+import {useBlogPost} from '@docusaurus/plugin-content-blog/client'
+
+export default function BlogPostItemWrapper(props) {
+  const { metadata } = useBlogPost()
+  const { frontMatter, slug, title } = metadata
+  const { comments = true } = frontMatter
+  return (
+    <>
+      <BlogPostItem {...props} />
+      {comments && (
+        <DiscussionEmbed
+          shortname='https-replica-io-dev'
+          config={{
+            url: slug,
```

Comment:
That should be the post's URL here, not slug. I think [`permalink`](https://github.com/facebook/docusaurus/blob/dd4fe63a3dfbdbfc264ef3f6bb93dbae6434d24b/packages/docusaurus-plugin-content-blog/src/plugin-content-blog.d.ts#L258) from [`metadata`](https://github.com/facebook/docusaurus/blob/dd4fe63a3dfbdbfc264ef3f6bb93dbae6434d24b/packages/docusaurus-plugin-content-blog/src/plugin-content-blog.d.ts#L245) is what should be used here.